src/components/Nav/Nav.scss

```diff
@@ -0,0 +1,25 @@
+header {
+  width:100%;
+  padding:10px 20px;
+  @include dropShadow;
+
+  .margin-wrap {
+    display: flex;
+    flex-flow:wrap;
+    justify-content: space-between;
+    align-items: center;
+    // padding:0px 20px;
```

Comment:
No need to change this now, but might want to delete the commented out padding at some point if it's unused.

Reply:
I went ahead and removed that commented out line and updated the PR.